test/pieces-children/pieces-malformed-child.js

```diff
@@ -0,0 +1,35 @@
+const t = require('../../test-lib/test.js');
+
+const apiKey = 'this is a test api key';
+
+(async function () {
+  setTimeout(function () {
+    process.exit(0);
+  }, 3000);
```

Comment:
A timeout is needed here to let initialization fail and an error be passed before killing the process. Otherwise the process exits to soon and the mocha test fails because of that. The timeout can be shorter if mocha testing alone, but needs slightly longer when running it with `nyc`.

Reply:
I see!
However I think we have an issue with the tests on the CI: https://github.com/apostrophecms/apostrophe/actions/runs/5056079387/jobs/9073107589#step:8:2530

<img width="737" alt="image" src="https://github.com/apostrophecms/apostrophe/assets/8301962/1bf43d27-1308-4b53-b34a-5c94cf83195f">